fxa/tests/test_oauth.py

```diff
@@ -602,6 +615,71 @@ def test_monkey_patch_for_gevent(self):
         fxa._utils.requests = old_requests
 
 
+class TestJwtToken(unittest.TestCase):
+
+    server_url = TEST_SERVER_URL
+
+    def callback(self, request):
+        if self.verify_success:
+            return (200, {}, self.body)
+        return (500, {}, "{}")
+
+    def setUp(self):
+        self.client = Client(server_url=self.server_url)
+        self.body = ('{"user": "alice", "scope": ["profile"],'
+                     '"client_id": "abc"}')
+        responses.add_callback(responses.POST,
+                               'https://server/v1/verify',
+                               callback=self.callback,
+                               content_type='application/json')
+        add_jwks_response()
+        self.verify_success = True
+
+    def get_file_contents(self, filename):
+        return jwt.algorithms.RSAAlgorithm.from_jwk(open(
+            os.path.join(
+                os.path.dirname(__file__),
+                filename
+            )
+        ).read())
+
+    @responses.activate
+    def test_good_jwt_token(self):
+        private_key = self.get_file_contents("private-key.json")
+        result = str(jwt.encode({}, private_key, "RS256", {"typ": "at+jwt"}))
+        self.client.verify_token(result)
```

Comment:
Please add an assertion here to check that we did **not** call the `/verify` endpoint in this case, since we were able to locally verify the token. I expect we can use the `responses.calls` list for this.